data/json/items/gun/3006.json

```diff
@@ -47,7 +46,6 @@
     "aim_speed" : 6,
     "recoil": -15,
     "durability": 8,
-    "reload": 150,
```

Comment:
Considering the gun's unusual magazine, I'm assuming the longer reload time was intended.